pom.xml

```diff
@@ -362,7 +363,7 @@
       <dependency>
         <groupId>org.jctools</groupId>
         <artifactId>jctools-core</artifactId>
-        <version>1.2</version>
+        <version>1.2.1</version>
```

Comment:
Can be updated to this constant directly:
https://github.com/JCTools/JCTools/blob/master/jctools-core/src/main/java/org/jctools/util/Pow2.java#L20